var/spack/repos/builtin/packages/opengl/package.py

```diff
@@ -53,6 +56,26 @@ def install(self, spec, prefix):
         In that case, /opt/opengl/ should contain these two folders:
 
         include/GL/       (opengl headers, including "gl.h")
-        lib               (opengl libraries, including "libGL.so")"""
+        lib               (opengl libraries, including "libGL.so")
+
+        On Apple Darwin (e.g., OS X, macOS) systems, this package is
+        normally installed as part of the XCode Command Line Tools in
+        /usr/X11R6, so a working packages.yaml would be
+
+        packages:
+          opengl:
+            paths:
+              opengl@4.1: /usr/X11R6
```

Comment:
I had to use `@4.3` in this line for this to work. Probably because the `provides(...)` commands above only list `4.3`-`4.5`.